src/helper/websocket/index.ts

```diff
@@ -85,7 +85,12 @@ export class WSContext<T = unknown> {
   }
 }
 
-export type WSMessageReceive = string | Blob | ArrayBufferLike
+export type WSMessageReceive =
```

Comment:
Replaced `ArrayBufferLike` with explicit types (`ArrayBuffer | SharedArrayBuffer | Uint8Array<ArrayBuffer>`) to support all buffer types.